manifests/09-prometheus-rules.yaml

```diff
@@ -28,3 +28,18 @@ spec:
           DeploymentConfigs which reference ImageStreamTags may not work as
           expected. Please configure storage and update the config to Managed
           state by editing configs.imageregistry.operator.openshift.io.
+  - name: ImagePruner
+    rules:
+    - alert: ImagePruningDisabled
+      expr: image_registry_operator_image_pruner_install_info < 2
```

Comment:
@brancz @metalmatze @simonpasquier perhaps `install_status` is a better metric name? This metric sits in between "metadata" info and a real value.

Reply:
Yeah that works as well. It’s true that info is really meant for constant value time series.